canvas_modules/common-canvas/src/common-properties/controls/expression/expression.jsx

```diff
@@ -353,12 +354,23 @@ class ExpressionControl extends React.Component {
 			</div>);
 		}
 
+		let toggleMaxMin = null;
+		if (this.props.control.enableMaximize) {
+			const isTearsheetOpen = this.props.controller.getActiveTearsheet() === this.props.control.data.tearsheet_ref;
```

Comment:
Does this have problems if `this.props.control.data` doesn't exist or isn't that possible in this case?